### DISCUSSION TITLE
Wanting to enable custom LED menu

### DISCUSSION THREAD

**Discussion by TheDanzilla** [KlipperScreen/KlipperScreen#1128] [Q&A]
Hi all,
I am wanting to create a custom LED menu within the More but when I amend the klipperscreen.conf it ignores the changes unless I amend the default.conf and comment out the existing LED menu and also the PIN menu options?

What am I doing wrong as I dont really want to amend the default.conf as update manager then reports a dirty repo unless I comment out klipperscreen from the moonraker update manager.

I did clone your repo to see if I could change the update repo path but that didnt work (im not a programer so have never done this before)

Any help would be appreciated.

**Comment by alfrix** [comment]
> when I amend the klipperscreen.conf it ignores the changes

show what changes are you making and the klipperscreen.log

https://klipperscreen.readthedocs.io/en/latest/Troubleshooting/

**Comment by TheDanzilla** [comment]
Attached is a few screen shots of the menu changes along with a zip containing my defaults.conf and my klipperscreen.conf

[KlipperScreen.zip](https://github.com/KlipperScreen/KlipperScreen/files/12743490/KlipperScreen.zip)
![20230928_073446](https://github.com/KlipperScreen/KlipperScreen/assets/137014006/4031df86-f0cc-49b5-ab49-4233e9d58076)
![20230928_073452](https://github.com/KlipperScreen/KlipperScreen/assets/137014006/34927d17-99ce-420b-97d5-24d5917f298b)
![20230928_073440](https://github.com/KlipperScreen/KlipperScreen/assets/137014006/23118a9d-3355-4624-b236-7cdd119be821)

**Comment by alfrix** [comment]
i see what you want to do, to replace the integrated panel you would have to unset the panel variable in klipperscreen.conf

```
[menu __main more led]
name: {{ gettext('LED') }}
icon: light
panel:
```
you have to update to the last commit for it to work

a note for further interactions, when i ask for the log, i want to see the log :(

**Reply by TheDanzilla** [reply]
appologies for not getting you the log, I wasnt sure how to pull the klipperscreen logs.

Thank you for the info, I will roll back to the last commit.
